File: app/src/main/java/com/serviceapps/shopping/ui/fragments/SellerOrdersBaseFragment.java
package com.serviceapps.shopping.ui.fragments;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.Nullable;
import androidx.viewpager.widget.ViewPager;

import com.google.android.material.tabs.TabLayout;
import com.serviceapps.shopping.R;
import com.serviceapps.shopping.ui.adapters.SellerOrderPagerAdapter;


public class SellerOrdersBaseFragment extends BaseFragment {

    View myFragment;

    ViewPager viewPager;
    TabLayout tabLayout;


    public SellerOrdersBaseFragment() {
        // Required empty public constructor
    }

    public static SellerOrdersBaseFragment getInstance()    {
        return new SellerOrdersBaseFragment();
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        myFragment = inflater.inflate(R.layout.seller_orders_base_layout, container, false);

        viewPager = myFragment.findViewById(R.id.viewPager);
        tabLayout = myFragment.findViewById(R.id.tabLayout);

        return myFragment;
    }

    //Call onActivity Create method


    @Override
    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
        super.onActivityCreated(savedInstanceState);

        setUpViewPager(viewPager);
        tabLayout.setupWithViewPager(viewPager);

        tabLayout.addOnTabSelectedListener(new TabLayout.OnTabSelectedListener() {
            @Override
            public void onTabSelected(TabLayout.Tab tab) {

            }

            @Override
            public void onTabUnselected(TabLayout.Tab tab) {

            }

            @Override
            public void onTabReselected(TabLayout.Tab tab) {

            }
        });
    }

    private void setUpViewPager(ViewPager viewPager) {
        SellerOrderPagerAdapter adapter = new SellerOrderPagerAdapter(getChildFragmentManager());

        adapter.addFragment(new OrdersFragment(), "Customer Orders");
        adapter.addFragment(new SellerOrdersFragment(), "Your Orders");

        viewPager.setAdapter(adapter);
    }
}

File: app/src/main/java/com/serviceapps/shopping/ui/fragments/SellerOrdersBaseFragment.kt
package com.serviceapps.shopping.ui.fragments

import com.serviceapps.shopping.ui.fragments.BaseFragment
import androidx.viewpager.widget.ViewPager
import com.google.android.material.tabs.TabLayout
import android.view.LayoutInflater
import android.view.ViewGroup
import android.os.Bundle
import android.view.View
import com.serviceapps.shopping.R
import com.google.android.material.tabs.TabLayout.OnTabSelectedListener
import com.serviceapps.shopping.ui.adapters.SellerOrderPagerAdapter
import kotlinx.android.synthetic.main.seller_orders_base_layout.*

class SellerOrdersBaseFragment : BaseFragment() {
    var myFragment: View? = null
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        myFragment = inflater.inflate(R.layout.seller_orders_base_layout, container, false)
        return myFragment
    }

    //Call onActivity Create method
    override fun onActivityCreated(savedInstanceState: Bundle?) {
        super.onActivityCreated(savedInstanceState)
        setUpViewPager(viewPager)
        tabLayout!!.setupWithViewPager(viewPager)
        tabLayout!!.addOnTabSelectedListener(object : OnTabSelectedListener {
            override fun onTabSelected(tab: TabLayout.Tab) {}
            override fun onTabUnselected(tab: TabLayout.Tab) {}
            override fun onTabReselected(tab: TabLayout.Tab) {}
        })
    }

    private fun setUpViewPager(viewPager: ViewPager?) {
        val adapter = SellerOrderPagerAdapter(childFragmentManager)
        adapter.addFragment(SellerOrdersFragment(), "Customer Orders")
        adapter.addFragment(OrdersFragment(), "Your Orders")
        viewPager!!.adapter = adapter
    }

    companion object {
        val instance: SellerOrdersBaseFragment
            get() = SellerOrdersBaseFragment()
    }
}